src/main/webapp/dev/js/main/instructorFeedbackEdit.es6

```diff
@@ -1087,7 +1087,11 @@ function readyFeedbackEditPage() {
     setupQuestionCopyModal();
 
     // Additional formatting & bindings.
-    disableEditFS();
+    if ($('#form_feedbacksession').data(`${ParamsNames.FEEDBACK_SESSION_ENABLE_EDIT}`) === true) {
```

Comment:
I know I originally suggested an optional JSP tag attribute, but I now feel it'd be better to make it required and always explicitly show it in our code. The thing that's optional should be the GET param, as that's displayed to the user right in the URL bar and can be easily modified.